### PR TITLE
behave fix clean up

### PR DESCRIPTION
**Pull request by meeuw**
## Description
In an earlier commit I've changed the current working directory and
the removal of a temporary file didn't take that into account.

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

File: test/features/steps/iocommands.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8
 from __future__ import unicode_literals
 import os
+import os.path
```
Comment (tsroten) [inline test/features/steps/iocommands.py]: `os.path` is automatically imported when `os` is imported (in line 3).

**Comment by tsroten** [comment]
Rather than all the `os.path.join` calls, why not just keep the code as it was before with `../` prefixed to the name. Then in the one place you need the filename, call `os.path.basename()`?

Then, the filename doesn't need to be a list anymore either, which is a little bit unintuitive.

**Comment by meeuw** [comment]
Hmm I didn't think of that, thanks!

File: test/features/steps/iocommands.py (hunk 1 above)
Comment (meeuw) [inline test/features/steps/iocommands.py]: right, thanks!

**Pull request merged**